examples/Services/AccountManagement/CreateCustomer.cs

```diff
@@ -0,0 +1,104 @@
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V0.Resources;
+using Google.Ads.GoogleAds.V0.Services;
+
+using System;
+
+namespace Google.Ads.GoogleAds.Examples.V0
+{
+    /// <summary>
+    /// This code example illustrates how to create a new customer under a given manager account.
+    ///
+    /// Note: this example must be run using the credentials of a Google Ads manager account.
+    /// By default, the new account will only be accessible via the manager account.
+    /// </summary>
+    public class CreateCustomer : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            CreateCustomer codeExample = new CreateCustomer();
+            Console.WriteLine(codeExample.Description);
+
+            long managerCustomerId = long.Parse("INSERT_MANAGER_CUSTOMER_ID_HERE");
+            codeExample.Run(new GoogleAdsClient(), managerCustomerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example illustrates how to create a new customer under a " +
+                    "given manager account.\n Note: this example must be run using the " +
+                    "credentials of a Google Ads manager account. By default, the new account " +
+                    "will only be accessible via the manager account.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="managerCustomerId">The manager customer ID.</param>
+        public void Run(GoogleAdsClient client, long managerCustomerId)
+        {
+            // Get the CustomerService.
+            CustomerServiceClient customerService = client.GetService(Services.V0.CustomerService);
+
+            Customer customer = new Customer()
+            {
+                DescriptiveName = $"Account created with CustomerService on '{DateTime.Now}'",
+
+                // For a list of valid currency codes and time zones see this documentation:
+                // https://developers.google.com/adwords/api/docs/appendix/codes-formats.
+                CurrencyCode = "USD",
+                TimeZone = "America/New_York",
+
+                // The below values are optional. For more information about URL
+                // options see: https://support.google.com/google-ads/answer/6305348.
+                TrackingUrlTemplate = "{lpurl}?device={device}",
+                FinalUrlSuffix = "keyword={keyword}&matchtype={matchtype}&adgroupid={adgroupid}",
```

Comment:
Same question as above.  Should these be injected variables, or is this syntax expected?